src/main/java/de/tum/in/www1/artemis/repository/OnlineUnitRepository.java

```diff
@@ -0,0 +1,13 @@
+package de.tum.in.www1.artemis.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.lecture.OnlineUnit;
+
+/**
+ * Spring Data JPA repository for the Online Unit entity.
+ */
+@Repository
+public interface OnlineUnitRepository extends JpaRepository<OnlineUnit, Long> {
+}
```

Comment:
More a question out of curiosity than a request for changes: Why do we need an empty repository class here?

Reply:
Ignore it, in context of the remaining code it is clear to me. 😅

Reply:
It's basically just a shorthand for `JpaRepository<OnlineUnit, Long>`.